waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/impl/PrefixBasedDatabaseMappingService.java

```diff
@@ -239,74 +247,95 @@ public DatabaseMapping databaseMapping(@NotNull String databaseName) {
     return mappings;
   }
 
-  private boolean isWhitelisted(String databasePrefix, String database) {
-    Whitelist whitelist = mappedDbByPrefix.get(databasePrefix);
-    if ((whitelist == null) || whitelist.isEmpty()) {
-      // Accept everything
-      return true;
-    }
-    return whitelist.contains(database);
-  }
+//  private boolean isWhitelisted(String databasePrefix, String database) {
```

Comment:
Unsure whether this should go into that MappingUtils class? If it's only used in this class then I'd say not....

Reply:
It's used in two other request classes, but I'll try to find a good place for this function :)